python/mxnet/test_utils.py

```diff
@@ -1915,8 +1915,8 @@ def chi_square_check(generator, buckets, probs, nsamples=1000000):
     if continuous_dist:
         sample_bucket_ids = sample_bucket_ids // 2
     obs_freq = np.zeros(shape=len(buckets), dtype=np.int)
-    for i in range(len(buckets)):
-        obs_freq[i] = (sample_bucket_ids == i).sum()
+    for i, _ in enumerate(buckets):
```

Comment:
Looks like you do not need to change this line. You just need i.

Reply:
nope, it fails on CI saying use_enumerate instead of range(Len())
Hence I had to put a separate commit c989695 for that